frontend/packages/console-shared/src/components/shortcuts/Shortcut.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { MouseIcon } from '@patternfly/react-icons';
+
+interface ShortcutProps {
+  children: React.ReactNode;
+  alt?: boolean;
+  click?: boolean;
+  ctrl?: boolean;
+  drag?: boolean;
+  hover?: boolean;
+  keyName?: string;
+  rightClick?: boolean;
+  shift?: boolean;
+}
+
+const Command: React.FC = ({ children }) => (
+  <span>
+    <kbd>{children}</kbd>
+  </span>
+);
+
+const Shortcut: React.FC<ShortcutProps> = ({
+  children,
+  ctrl,
+  shift,
+  alt,
+  keyName,
+  drag,
+  click,
+  rightClick,
+  hover,
+}) => {
+  const isMac = window.navigator.platform.includes('Mac');
+  return (
+    <tr>
+      <td>
+        {!isMac && ctrl && <Command>Ctrl</Command>}
+        {alt && <Command>{isMac ? '⌥ Opt' : 'Alt'}</Command>}
+        {shift && <Command>Shift</Command>}
+        {isMac && ctrl && <Command>⌘ Cmd</Command>}
```

Comment:
Any reason why we don't group `ctrl` and `cmd` together?

Reply:
on mac the trend is to put command before the key and after other modifiers:
![image](https://user-images.githubusercontent.com/14068621/69256750-f3620800-0b87-11ea-99ab-f6ad9a82dde0.png)

Reply:
Ah, good call.